crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -80,6 +83,10 @@ pub async fn import(
     };
     let private_key = &SigningKey::from_secret_scalar(*private_key);
 
+    let account_name = account
+        .clone()
+        .unwrap_or_else(|| generate_account_name(accounts_file).unwrap());
```

Comment:
I'm using `unwrap_or_else` here because, `unwrap_or` is eagerly compiled and since we were using the return of a function call in the `unwrap_or`, it means, the function will be evaluated even when we don't need the value.